src/clj/rems/db/users.clj

```diff
@@ -69,7 +70,11 @@
        (map :userid)
        (doall)))
 
-(defn get-user [userid]
-  (->> userid
-       get-user-attributes
-       format-user))
+(defn get-user
+  "Given a userid, returns a map with keys :userid, :mail and :name."
+  [userid]
+  (-> userid
+      get-user-attributes
+      format-user
+      ;; hack for users without attributes
+      (assoc :userid userid)))
```

Comment:
I'm not sure offhand what this is needed for; can you clarify in the comment above?

Reply:
get-user-attributes returns nil for unknown users, so we get `{:userid nil :name nil :email nil}` from format-user

Reply:
Right, I mean overall: is this function ever legitimately called for non-existent user (e.g., in the case of a member invited from outside REMS)? If it is, could you add a clarification to the comment on L79 so that the purpose of having this hack would be obvious? Thanks!